src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  position: absolute;
```

Comment:
Actually you don't need to use `absolute `for such simple layout, just use `padding `property to set paddings between bars

Reply:
Do you mean like I do not need absolute in my entire CSS? 
(...'cause I used absolute here in this form just to place input inside its parent block.)

Reply:
![image](https://user-images.githubusercontent.com/122989585/214016676-bca1ac80-190e-4e38-9c65-d8aa1f0a24a1.png)

If you have more questions ask them in chat next time pls)

Reply:
only one place where you need the absolute value is `form__search-button`. For that you can set `position: relative;` to the form. but no need to position absolute forms and inputs